app/routes/logout.tsx

```diff
@@ -0,0 +1,17 @@
+import { ActionFunction, LoaderFunction, redirect } from "remix";
+import { logout } from "../../app/utils/auth.server";
```

Comment:
Since we already have something to change, let's tidy this up while we're at it.
```suggestion
import { logout } from "~/utils/auth.server";
```